SpatialGDK/Source/SpatialGDK/Private/Utils/InterestFactory.cpp

```diff
@@ -382,4 +381,24 @@ QueryConstraint InterestFactory::CreateLevelConstraints() const
 	return LevelConstraint;
 }
 
+TArray<uint32> InterestFactory::CreateClientResultType() const
+{
+	TArray<uint32> ResultType;
+
+	// Add the required unreal components
+	ResultType.Add(SpatialConstants::UNREAL_METADATA_COMPONENT_ID);
+	ResultType.Add(SpatialConstants::SPAWN_DATA_COMPONENT_ID);
+	ResultType.Add(SpatialConstants::NETMULTICAST_RPCS_COMPONENT_ID_LEGACY);
+	ResultType.Add(SpatialConstants::MULTICAST_RPCS_COMPONENT_ID);
+	ResultType.Add(SpatialConstants::RPCS_ON_ENTITY_CREATION_ID);
+
+	// TODO: breaks without this, but shouldn't need it. Figure out why
```

Comment:
How does it break?

Reply:
Can you create a tracking Jira ticket please?

Reply:
done, and mentioned in comment